src/managers/quorum.py

```diff
@@ -176,7 +177,10 @@ def update_acls(self) -> None:
 
             # FIXME: data-platform-libs should handle this when it's implemented
             if client.chroot:
-                requested_chroots.add(client.chroot)
+                if event and client.relation and client.relation.id == event.relation.id:
+                    continue  # skip broken chroots, so they're removed
```

Comment:
Does the removal happen automatically?

Reply:
I chose to not remove just the `chroot` parent, but to remove all nodes recursively from it, with the `chroot` as the last one.